webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleViewHandler.java

```diff
@@ -223,12 +223,12 @@ protected void wrappedService(HttpServletRequest request, HttpServletResponse re
             }
             query = parseQuery(body.toString());
 
-            String json = query.get("sheets");
+            String json = query.get("sheets"); // all the scrambles
```

Comment:
Why do we need to pull out this json string and pass it down? Can we instead use the parsed `scrambleRequests` variable? I see that we currently are just reparsing this in OrderedScrambles.java, and that feels redundant to me.

Reply:
I just added the //comment while I was trying to understand the flux from front-end to the back-end. I'd rather not to change that.

Reply:
Nah, fixed.